src/printer.ts

```diff
@@ -787,7 +788,7 @@ export class PugPrinter {
 		formatEmptyAttribute(token, this.options.pugEmptyAttributes, this.options.pugEmptyAttributesForceQuotes);
 
 		if (typeof token.val === 'string') {
-			if (isQuoted(token.val)) {
+			if (isQuoted(token.val) && !isBacktickQuoted(token.val)) {
```

Comment:
@Yann-prak Do you think this is a good idea?
Do you think there is a better way?